packages/rum-core/src/domain/rumEventsCollection/action/getSelectorsFromElement.ts

```diff
@@ -105,6 +125,10 @@ function getClassSelector(element: Element): string | undefined {
   }
 }
 
+function getTagNameSelector(element: Element): string {
```

Comment:
<!-- Suggestions are specific requests to improve the subject under review. It is assumed that we all want to do what's best, so these comments are never dismissed as “mere suggestions”, but are taken seriously.  -->
**💬 suggestion:** ‏If we are making a `getTagNameSelector()` it could worth:

- updating `getPositionSelector()` so that it return `undefined` instead of `element.tagName` when there is no siblings 
- add `getTagNameSelector()` to the `uniqueAmongChildrenSelectorStrategies`